CONTRIBUTING.rst

```diff
@@ -24,42 +22,117 @@ If you are proposing a feature:
 
 
 Get Started!
-------------
+============
 
-Ready to contribute? Here's how to set up `earthpy` for local development.
+Ready to contribute? Here's how to set up EarthPy for local development.
 
-1. Fork the `earthpy` repo on GitHub.
-2. Clone your fork locally::
+1. Fork the repository on GitHub
+--------------------------------
 
-    $ git clone git@github.com:your_name_here/earthpy.git
+To create your own copy of the repository on GitHub, navigate to the
+`earthlab/earthpy <https://github.com/earthlab/earthpy>`_ repository
+and click the **Fork** button in the top-right corner of the page.
+
+2. Clone your fork locally
+--------------------------
 
-3. Set up your fork for local development with conda::
+Use ``git clone`` to get a local copy of your EarthPy repository on your
+local filesystem::
 
+    $ git clone git@github.com:your_name_here/earthpy.git
     $ cd earthpy/
+
+3. Set up your fork for local development
+-----------------------------------------
+
+Create an environment
+^^^^^^^^^^^^^^^^^^^^^
+
+Using conda, there are two options.
+
+1. The first and easiest option is to create an environment from the
+``environment.yml`` file.
+Note that this will only allow you to test against one version of python
+locally, but this is the recommended option on Windows::
+
     $ conda env create -f environment.yml
     $ source activate earthpy-dev
```

Comment:
Heya @mbjoseph so interestingly we learned yesterday that the new versions of conda all use `conda activate` rather than `source activate` ALSO this works on windows EVEN in command prompt after testing :) however i am not sure how far back it goes ie whether older versions of conda support it but apparently they are deprecating this.  [More here](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)

Reply:
Thanks @lwasser - I have updated the contributing file to reflect the `conda activate` syntax